File: Readio/src/main/java/ninegle/Readio/mypage/mapper/MyPageUserMapper.java
package ninegle.Readio.mypage.mapper;

import ninegle.Readio.mypage.dto.response.UserInfoDto;
import ninegle.Readio.user.domain.User;

public class MyPageUserMapper {

	public static UserInfoDto toUserInfoDto(User user) {
		return new UserInfoDto(
			user.getEmail(),
			user.getNickname(),
			user.getPhoneNumber(),
			user.getPoint()
		);
	}
}

File: Readio/src/main/java/ninegle/Readio/mypage/mapper/MyPageUserMapper.kt
package ninegle.Readio.mypage.mapper

import ninegle.Readio.mypage.dto.response.UserInfoDto
import ninegle.Readio.user.domain.User
object MyPageUserMapper {
	fun toUserInfoDto(user: User): UserInfoDto {
		return UserInfoDto(
			email = user.email,
			nickname = user.nickname,
			phoneNumber = user.phoneNumber,
			point = user.point
		)
	}
}